utils/utils.py

```diff
@@ -128,25 +141,18 @@ def generate_services():
     Also assigns allowed data fields for each created service.
     """
     services = []
-    for service_type in ServiceType:
-        service = Service.objects.filter(service_type=service_type).first()
+    for service_spec in SERVICES:
+        service_name = service_spec["name"]
+        service = Service.objects.filter(name=service_name).first()
```

Comment:
Same filter() vs get() here. In my opinion when getting one explicit thing it's more idiomatic Django to use
```python
try:
    instance = Model.objects.get(key=value)
except Model.objects.DoesNotExist:
    instance = Model(...)
```

And maybe handle MultipleObjectsReturned exception where appropriate.

Reply:
Changed